directLinking.js

```diff
@@ -1,26 +1,37 @@
 "use strict";
 
 const fetch = require('node-fetch');
+const logger = require('./logger');
 
 const API_URL_ORIGIN = 'https://integration-api.securecodewarrior.com';
 const API_URL_PATH = '/api/v1/trial';
 const PARTNER_ID = 'github-sarif-action';
 
 async function getTrainingData(mappingListId, mappingKey, languageKey) {
+
+    // create an list of values to populate into the Id param of the DI linking API
+    let idValue = [PARTNER_ID];
+    if (process.env.GITHUB_REPOSITORY) {
+        idValue.push(process.env.GITHUB_REPOSITORY.split('/')[0]);
```

Comment:
I would suggest extracting `process.env.GITHUB_REPOSITORY.split('/')[0]` to an explanatory const, eg `githubOwner`, just for clarity.